### PR TITLE
Fix typos and grammar in DES manpage

### PR DESCRIPTION
**Pull request by wesinator**
Clean up typos and grammar in the man3 DES section

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/DES_random_key.pod [hunk 1]
```diff
@@ -181,31 +181,31 @@ chaining mode used by Kerberos v4. Its parameters are the same as
 DES_ncbc_encrypt().
 
 DES_cfb_encrypt() encrypt/decrypts using cipher feedback mode.  This
```
Comment (mspncp) [inline doc/man3/DES_random_key.pod]: While your at it, please replace `encrypt/decrypts  ->  encrypts/decrypts`

File: doc/man3/DES_random_key.pod [hunk 2]
```diff
@@ -181,31 +181,31 @@ chaining mode used by Kerberos v4. Its parameters are the same as
...
 be passed to the next call to this function.  Since this function runs
 a complete DES ECB encryption per I<numbits>, this function is only
-suggested for use when sending small numbers of characters.
+suggested for use when sending a small number of characters.
 
 DES_cfb64_encrypt()
-implements CFB mode of DES with 64bit feedback.  Why is this
+implements CFB mode of DES with 64-bit feedback.  Why is this
 useful you ask?  Because this routine will allow you to encrypt an
 arbitrary number of bytes, no 8 byte padding.  Each call to this
```
Comment (mspncp) [inline doc/man3/DES_random_key.pod]: `... , no 8 byte padding.  -> ... without 8 byte padding.`

File: doc/man3/DES_random_key.pod [hunk 3]
```diff
@@ -170,42 +170,42 @@ of 24 bytes.  This is much better than CBC DES.
...
 DES_cfb64_encrypt()
-implements CFB mode of DES with 64bit feedback.  Why is this
+implements CFB mode of DES with 64-bit feedback.  Why is this
 useful you ask?  Because this routine will allow you to encrypt an
-arbitrary number of bytes, no 8 byte padding.  Each call to this
+arbitrary number of bytes, without 8 byte padding.  Each call to this
 routine will encrypt the input bytes to output and then update ivec
 and num.  num contains 'how far' we are though ivec.  If this does
-not make much sense, read more about cfb mode of DES :-).
+not make much sense, read more about CFB mode of DES :-).
```
Comment (paulidale) [inline doc/man3/DES_random_key.pod]: While this is being changed, can we lose the :-) at the end too?

**Comment by mspncp** [comment]
@wesinator we don't have a signed CLA from you. Since your contribution qualifies as trivial, we can merge it if you add the `CLA: trivial` statement as a separate line to the commit body (which needs to be separated from the title by an empty line). You can do this either by amending your last commit, or rebasing and squashing your branch. (It will be squashed in the end anyway by me either way.)

Since it looks to me that you did the editing via web editing and you might not have a local copy to do that, I would accept if you post a comment saying "I a gree it's a trivial change", in which case I would add the statement to the body for you when merging.

If you intend to contribute more in the future then the simplest solution would be to send us a signed CLA, see https://www.openssl.org/policies/cla.html for more details. Note that your personal email address is hidden, so unless you change that the CLA needs to be signed for your GitHub address.

**Comment by wesinator** [comment]
I agree it's a trivial change.

Thanks

**Comment by mspncp** [comment]
I just force-pushed a rebased and squashed version of your pull request with the `CLA: trivial` statement to your repository.

**Comment by mspncp** [comment]
Although not really necessary, I'll wait for the CI to finish before merging.

**Comment by mspncp** [comment]
Merged, thank you!

4fef4981f8 (master)
521738e990 (OpenSSL_1_1_1-stable)

**Pull request closed without merging**